### PR TITLE
Remove Battery Devices from *_All Commands

### LINKED ISSUE #533: Battery Sensor Devices Should not Be Included in *_All Commands
This applies to sync_all, refresh_all, join_all and any others

I think our initial thought had been that these messages would be cached for battery devices, but that doesn't seem to work out correctly.  I have to revisit why I concluded this isn't working, but the solution was just to skip battery devices in the all commands.

### PR DESCRIPTION
**Pull request by krkeegan**
## Proposed change
I remember why this was an issue and have updated issue #533.

In short battery devices are present, the sequence for the *_all command will halt when it gets to a battery device and only resume when that device has woken up and the command was sent.

This is at best confusing to the user, because they expect to see the sequence run to completion and receive a message like
"Refresh All Devices Complete"

At worst, this is problematic if a device after a battery device in the sequence is not getting the proper command and the user
assumes that it is.

_Note_
There is an alternative solution in which the commands are sent immediately to battery devices with the assumption that they are asleep and the command will be queued.  This seems bad to me as the devices may be awake, in which case they could collide with each other.  Also the messages (other than `refresh` for which there is a test) would just pile up in the battery device's queue potentially with numerous duplicates if these were called a lot.

Overall, I think they best option is for users to call these commands on each desired battery device directly.

## Additional information
- This PR fixes or closes issue: fixes #533

## Checklist
- [x] The code change is tested and works locally.
- [x] Local tests pass.
- [x] Code documentation was added where necessary

If user exposed functionality or configuration variables are added/changed:

- [x] Documentation added/updated

### CODE REVIEW

File: insteon_mqtt/Modem.py [hunk 1]
```diff
@@ -478,7 +486,8 @@ def get_engine_all(self, on_done=None):
 
         # Reload all the device databases.
```
Comment (tstabrawa) [inline insteon_mqtt/Modem.py:487]: I might be worth it to update this comment while we're in here.  Looks like it was copy-pastad from refresh-all.

```suggestion
        # Run Get Engine on all the devices
```

**Review by tstabrawa — APPROVED** [review]

**Comment by krkeegan** [comment]
This is also working fine on my local instance.
